TeacherWorkout.Api/Startup.cs

```diff
@@ -70,25 +66,10 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, TeacherW
 
             db.Database.Migrate();
 
-            app.UseGraphQL<ISchema>();
             app.UseRouting();
 
-            app.UseAuthorization();
-
-            app.UseEndpoints(endpoints =>
-            {
-                endpoints.MapControllers();
-            });
-        }
-
-        private static void AddGraphQLNamespaces(IServiceCollection services)
-        {
-            AppDomain.CurrentDomain.GetAssemblies()
-                .SelectMany(t => t.GetTypes())
-                .Where(t => t.IsClass)
-                .Where(t => t.Namespace != null && t.Namespace.StartsWith("TeacherWorkout.Api.GraphQL.Types"))
-                .ToList()
-                .ForEach(t => services.AddSingleton(t));
```

Comment:
Why remove this?

Reply:
As far as I understand, this method is used to register the GraphQL schema types with the dependency injection container. However, GraphQL.NET already offers the [`AddGraphTypes` method](https://github.com/graphql-dotnet/server/blob/2ab8faaf71b5ec4a6075a9459c28436bcd14b881/src/Core/Extensions/GraphQLBuilderCoreExtensions.cs#L93) (used above) for adding all the types implementing `IGraphType` in the calling assembly.

Reply:
:+1: